### PR TITLE
Remove lines to hardcode contents to use JSONWidget

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,6 +2,6 @@ async_lru~=1.0
 galaxy_importer>=0.3.1
 jsonschema>=3.0
 packaging
-pulpcore>=3.15.0.dev
+pulpcore>=3.15.1
```
Comment (fao89) [inline requirements.txt:5]: I think it will break the CI even when you add "Required PR"

Comment (daviddavis) [reply requirements.txt:5]: I think it runs against 3.16.0.dev (which is >= 3.15.1) and thus works?

Comment (fao89) [reply requirements.txt:5]: yeah, you are right, I thought pulpcore master was 3.15.1.dev, but it is 3.16.0.dev

**Review by fao89 — APPROVED** [review]
thank you!

**Pull request merged**